draft-ietf-acme-acme.md

```diff
@@ -411,10 +412,25 @@ For all other requests, the request is signed using an existing account and
 there MUST be a "kid" field. This field MUST contain the account URL received by
 POSTing to the newAccount resource.
 
-Note that authentication via signed JWS request bodies implies that GET requests
-are not authenticated.  Servers MUST NOT respond to GET requests for resources
-that might be considered sensitive.  Account resources are the only sensitive
-resources defined in this specification.
+Note that authentication via signed JWS request bodies implies that
+requests other than POST requests are not authenticated.  The server
+MUST allow GET requests for its directory resource and both GET and
+HEAD requests for its newNonce resource (see {{resources}}), in
+order to support client bootstrapping.
+
+Otherwise, if a client wishes to fetch a resource from
+the server (which would otherwise be done with a GET), then it MUST
+send a POST request with a JWS body as described above, with the
+following differences:
+
+* The JWS Protected Header MUST contain a "typ" field with the value
+  "GET"
+* The payload MUST be an empty JSON object ({})
```

Comment:
What should the server return to the client if this isn't true? A `urn:ietf:params:acme:error:unauthorized` type problem? `urn:ietf:acme:error:malformedRequest`?